packages/zoe/src/zoeService/zoe.js

```diff
@@ -58,235 +59,53 @@ function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
     invitationIssuer,
   } = makeZoeStorageManager(createZCFVat);
 
-  const {
-    getInstance,
-    getInstallation,
-    getInvitationDetails,
-  } = makeInvitationQueryFns(invitationIssuer);
+  // Pass the capabilities necessary to create zoe.startInstance
+  const startInstance = makeStartInstance(
+    zoeServicePromiseKit.promise,
+    makeZoeInstanceStorageManager,
+    unwrapInstallation,
+  );
 
+  // Pass the capabilities necessary to create zoe.offer
   const offer = makeOffer(
     invitationIssuer,
     getInstanceAdmin,
     depositPayments,
     getAssetKindByBrand,
   );
 
+  // Make the methods that allow users to easily and credibly get
+  // information about their invitations.
+  const {
+    getInstance,
+    getInstallation,
+    getInvitationDetails,
+  } = makeInvitationQueryFns(invitationIssuer);
+
   /** @type {ZoeService} */
   const zoeService = Far('zoeService', {
-    getInvitationIssuer: () => invitationIssuer,
     install,
+    startInstance,
```

Comment:
These methods are reordered so that we can distinguish between methods that are getters and methods that change state.

Reply:
Good. Resolving.